### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by pvieira84** [oss-apps/split-pro#88] [Ideas]
Any chance an API can be added to the roadmap? I believe it would be helpful and handy for a large group of people

**Comment by KMKoushik** [comment]
i like the idea of having apis. currently the whole project is written in trpc, so can be used with a trpc client. But really need some help for proper rest apis

**Comment by pvieira84** [comment]
Unfortunately I don't have experience on these tecnologies hence won't be able to help you on this but I believe it would be a great addiction to this tool

**Comment by KMKoushik** [comment]
@pvieira84 agreed, sadly no bandwidth for this atm, might need some help from the community!
